### PR TITLE
Add LICENSE file

### PR DESCRIPTION
**Pull request by stepps00**
Will fix https://github.com/whosonfirst-data/whosonfirst-data/issues/1795.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ Repository examples:
 Crediting Who's On First is recommended and linking back to this License is required.
 
 ```
-Data from Who's On First. <a href="https://whosonfirst.org/docs/licenses/">License</a> and <a href="https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/README.md">Sources</a>.
+Data from Who's On First <a href="https://whosonfirst.org/docs/sources/list/">sources</a>.
```
Comment (nvkelso) [inline README.md]: > Data from Who's On First. <a href="https://whosonfirst.org/docs/sources/list/">License</a>.

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# License
+
+Crediting Who's On First is recommended and linking back to the License is required. For example:
+
+> Data from Who's On First. [License](https://whosonfirst.org/docs/licenses/).
+
+The Who's On First dataset is both original work and a modification of existing open data. Some of those open data projects do require attribution. We have listed all sources in the full [license file](https://whosonfirst.org/docs/licenses/).
```
Comment (nvkelso) [inline LICENSE.md]: `an original work` (add `an`)

Comment (stepps00) [reply LICENSE.md]: fixed in 4171473

File: README.md (hunk 1 above)
Comment (stepps00) [reply README.md]: fixed in 4171473

**Pull request merged**